docs/hosts.md

```diff
@@ -339,6 +339,15 @@ non-compliant OCI registries which are missing the `/v2` prefix.
 override_path = true
 ```
 
+## dial_timeout field
+
+`dial_timeout` is the maximum amount of time a dial will wait for
+a connect to complete.
```

Comment:
Can you be clear what the default the is and the use case for this. I assume the primary use case for this is to make it much much shorter for some mirrors